src/test/scala/chiselTests/ConstSpec.scala

```diff
@@ -0,0 +1,62 @@
+package chiselTests
+
+import chisel3._
+import chiselTests.{ChiselFlatSpec, Utils}
+import chisel3.experimental.BundleLiterals.AddBundleLiteralConstructor
+import chisel3.experimental.VecLiterals.AddVecLiteralConstructor
```

Comment:
Are these used? Are they supposed to be?

Reply:
oh whoops, i added them to support bundle/vec init with const, but ended up taking that out of this PR

Reply:
should i open up another PR to take out these imports?